.github/workflows/pre-merge.yml

```diff
@@ -0,0 +1,47 @@
+---
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ###########################
+  #  BUILD STAGE
+  ###########################
+  build-kotlin:
+    runs-on: infra1-medium
+#    needs: compliance-check
```

Comment:
Can be removed. Compliance will be solved purely using repository configuration.